package.json

```diff
@@ -39,5 +39,8 @@
     "url": "https://zouhir.org/"
   },
   "license": "MIT",
-  "repository": "https://github.com/foldable-devices/device-configurator"
+  "repository": "https://github.com/foldable-devices/device-configurator",
```

Comment:
This line is unrelated, but looks wrong

Reply:
You can just fix it :) by replacing it with "https://github.com/foldable-devices/spanning-css-polyfill.git"